docs/tutorials/get_started.ipynb

```diff
@@ -0,0 +1,247 @@
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit import QuantumCircuit\n",
+    "\n",
+    "# Create a new circuit with two qubits (first argument) and two classical\n",
+    "# bits (second argument)\n",
+    "qc = QuantumCircuit(2, 2)\n",
+    "\n",
+    "# Add a Hadamard gate to qubit 0\n",
+    "qc.h(0)\n",
+    "\n",
+    "# Perform a controlled-X gate on qubit 1, controlled by qubit 0\n",
+    "qc.cx(0, 1)\n",
+    "\n",
+    "# Measure qubit 0 to cbit 0, and qubit 1 to cbit 1\n",
+    "qc.measure(0, 0)\n",
+    "qc.measure(1, 1)\n",
+    "\n",
+    "# Return a drawing of the circuit using MatPlotLib (\"mpl\"). This is the\n",
+    "# last line of the cell, so the drawing appears in the cell output.\n",
+    "# Remove the \"mpl\" argument to get a text drawing.\n",
+    "qc.draw(\"mpl\")"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "0c957de9",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "See :class:`~.QuantumCircuit` in the documentation for all available operations."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9acac1d4",
+   "metadata": {},
+   "source": [
+    "## Run your quantum program\n",
+    "\n",
+    "Quantum circuits can produce random results, so you'll often want to collect a sample of the outputs by running the circuit many times. You can sample a circuit's output using the `Sampler` class. \n",
+    "\n",
+    "This tutorial uses Qiskit's built-in `Sampler`, which simulates a perfect quantum computer, but can only handle small numbers of qubits. You can replace this with samplers of higher-performance simulators from [Qiskit Aer](https://github.com/Qiskit/qiskit-aer#readme), or with samplers that send your circuit over the internet to run on real quantum computers (such as [IBM's Qiskit Runtime](https://quantum-computing.ibm.com/lab/docs/iql/runtime/) sampler)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "69a8d872",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "SamplerResult(quasi_dists=[{0: 0.4999999999999999, 3: 0.4999999999999999}], metadata=[{}])"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.primitives import Sampler\n",
+    "\n",
+    "# Create a Sampler object\n",
+    "sampler = Sampler()\n",
+    "\n",
+    "# Submit the circuit to the sampler\n",
+    "job = sampler.run(qc)\n",
+    "\n",
+    "# Once the job is complete, get the result\n",
+    "job.result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dc5ce1eb",
+   "metadata": {},
+   "source": [
+    "The `qasi_dists` property is a list of sampled distributions for each circuit we provided. Since we only provided one circuit, the list only has one element.\n",
```

Comment:
```suggestion
    "The `quasi_dists` property is a list of sampled distributions for each circuit we provided. Since we only provided one circuit, the list only has one element.\n",
```